src/audiomixerboard.cpp

```diff
@@ -807,13 +817,27 @@ void CAudioMixerBoard::ChangeFaderOrder ( const bool        bDoSort,
     }
 }
 
+QString CAudioMixerBoard::GetTitle()
+{
+    return "[" + csRecorderStateToString ( eRecorderState ) + "] " + tr ( "Personal Mix at the Server: " ) + strServerName;
```

Comment:
I would prefer that if the recorder does not record, no additional string is show. Just in case the recorder actually records, a clear message apears, preferably in capital letters like "RECORDING ACTIVATED".

Reply:
Done.